images/pulp/stable/Containerfile

```diff
@@ -12,7 +12,9 @@ ARG PULP_MAVEN_VERSION=""
 ARG PULP_PYTHON_VERSION=""
 ARG PULP_RPM_VERSION=""
 ARG PULP_OSTREE_VERSION=""
+ARG PULP_UI_URL=""
 
+ENV PULP_UI=true
```

Comment:
This should probably depend on PULP_UI_URL... Or let's just make PULP_UI_URL the env var?